app/frontend/src/styles/application/form.scss

```diff
@@ -67,6 +67,21 @@
   }
 }
 
+.inline-form-group {
+  display: inline-block;
+
+  &.inline-grade {
+    $width: govuk-grid-width(one-third);
+    $padding-left: govuk-spacing(3);
+    padding-left: $padding-left;
+    width: calc(#{$width} - #{$padding-left});
```

Comment:
not sure exactly the use case of this calculation but can you use flex(box) to acheive this?

Reply:
not that using calc is wrong but i try to use them as a last resort